.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -17,6 +17,10 @@ on:
         required: true
         default: "true"
   workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: 'A token passed from the caller workflow'
+        required: true
```

Comment:
this means that running this manually, I need to find the token somewhere and provide it?

Reply:
No this is for the gha that calls this action. In the L59 there is logic for that, the user doesn't have to deal with the secrets.